src/dash-table/dash/DataTable.js

```diff
@@ -1114,6 +1148,10 @@ export const propTypes = {
         PropTypes.oneOfType([PropTypes.string, PropTypes.number])
     ),
 
+    /**
+     * `derived_viewport_selected_columns` contains the ids of the
+     * `selected_columns` that are not currently hidden.
```

Comment:
Looks a bit funny as a double negative... though I guess this way is technically the most correct because a column can be "not currently hidden" but also not visible if it's scrolled off-screen.